tests/unit/lms/services/jstor/service_test.py

```diff
@@ -168,28 +174,24 @@ def test_thumbnail_handles_404s(self, svc, http_service, response, exception):
             svc.thumbnail("article_id")
 
     @pytest.fixture
-    def http_service(self, http_service):
-        http_service.get.return_value = factories.requests.Response(
-            raw="https://s3.example.com/pdf"
```

Comment:
It is interesting that eliminating this fake response does not cause any tests to fail. Presumably that is a mistake elsewhere.

Reply:
Yeah. Some of our fixtures are also weird and don't return mocks. Not sure what was happening here.